### PR TITLE
My commit with siameseCat from Kholodkova Nataliya

### PR DESCRIPTION
**Pull request by UA-girl**
First try )

### CODE REVIEW

**Review by babayK0 — APPROVED** [review]
Nice! :+1:

File: README.md [hunk 1]
```diff
@@ -40,6 +40,11 @@ P.S. Where all postings before March 2018 have gone to?
 
 ---
 
+## Hi Kottans ))
+I'm Nataliya and these are my first steps in FrontEnd. I realy hope that everyone of us will get his (or her :) ) dream )
+[![SiameseCat](./assets/images/old_fashioned_siamese_cat_with_amazing_eyes.jpg)](https://github.com/kottans/frontend)
```
Comment (babayK0) [inline README.md:45]: Fix typo please

`[![SiameseCat]`

**Review by babayK0 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (UA-girl) [reply README.md]: Fixed ))) Is it ok, or I don't see something else wrong? )

Comment (lenadgit) [reply README.md]: fixed. nice job)

**Review by lenadgit — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (UA-girl) [reply README.md]: Thank you!

**Pull request merged**